portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for creating new comments. */
+@WebServlet("/new-comment")
+public class NewCommentServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    String name = getParameter(request, "user-name", "Anonymous");
+    String text = getParameter(request, "user-text", "");
```

Comment:
To make this easier to maintain, consider separating the logic that deals with Datastore from the logic that handles the PostRequest through a new function. 

One other benefit is that if you decide to use a different database (e.g. Firebase, MongoDB) later on you would only need to update the function related to writing to the database.

Reply:
Good idea - I made methods `writeToDatabase()` and `getFromDatabase()` to separate this logic out.